src/page-analysis/page-metadata-rules.ts

```diff
@@ -1,35 +1,36 @@
-import { metadataRules } from 'page-metadata-parser'
+import { metadataRuleSets } from 'page-metadata-parser'
 
 /**
  * Collection of fathom rules to tell `page-metadata-parser` where to extract certain metadata from
  */
 
 // We only want to fallback to the opengraph URL
 const canonicalUrl = {
-    ...metadataRules.url,
+    ...metadataRuleSets.url,
+    defaultValue: undefined,
```

Comment:
Default value for `url` rules set was added here: https://github.com/mozilla/page-metadata-parser/commit/920b78650099a7254cb5218737f4c5d4fe3a511f#diff-09461573a85e2d94f056dd6814769042R148